app/Services/MeetingService.php

```diff
@@ -80,9 +94,15 @@ public function start(): bool
             ->setLockSettingsDisableNote($this->meeting->room->lock_settings_disable_note)
             ->setLockSettingsHideUserList($this->meeting->room->lock_settings_hide_user_list)
             ->setLockSettingsLockOnJoin($this->meeting->room->lock_settings_lock_on_join)
-            ->setMuteOnStart($this->meeting->room->mute_on_start)
-            ->setMeetingLayout(MeetingLayout::CUSTOM_LAYOUT)
-            ->setLearningDashboardEnabled(false);
+            ->setMuteOnStart($this->meeting->room->mute_on_start);
+
+        if (is_null($meetingParams->getMeetingLayout())) {
```

Comment:
Please add a deprecation notice to these lines, add add it to the changelog, that we no longer force learingDashboard disabled and the custom layout, users should add it as a custom createParameter in the room type settings